connectivity/mqtt/req_doc.md

```diff
@@ -0,0 +1,49 @@
+## Instruction
+The format of MQTT request is the same as http request, but MQTT request has one more field is `Device ID`.
+
+## Topic vs API table
+
+| topic                   | API                                | HTTP Method   |
+| -------------           |:-------------:                     | -------------:|
+| address                 | api_generate_address               | GET
```

Comment:
Is the trailing `|` missing?

Reply:
I examined the github browser, and it looks fine.